examples/gno.land/r/demo/tests/tests_test.gno

```diff
@@ -22,14 +22,10 @@ func TestAssertOriginCall(t *testing.T) {
 			t.Errorf("expected panic with '%v', got '%v'", expectedReason, r)
 		}
 	}()
-	func() {
-		// if called inside a function literal, this is no longer an origin call
-		// because there's one additional frame (the function literal).
-		if IsOriginCall() {
-			t.Errorf("expected IsOriginCall=false but got true")
-		}
-		AssertOriginCall()
-	}()
```

Comment:
No longer need to add an other frame (the function literal) to cheat the `AssertOriginCall` algorithm, since it doesn't use the number of frames any more.